CHANGES.md

```diff
@@ -1,3 +1,39 @@
+## edge-19.9.4
+
+This edge release introduces support for distributed tracing as well as a
+redesigned sidebar in the Web UI!
+
+Support for distributed tracing means that Linkerd data plane proxies can now
+emit trace spans, allowing you to see the exact amount of time spent in the
+Linkerd proxy for traced requests. The new `config.linkerd.io/trace-collector`
```

Comment:
Nit: Is it the amount of time spent in the proxy, or in the actual service?

Reply:
if you have tracing then you already know the amount of time spent in the service.  This change allows you to see the time spent in the proxy as well.